applications/nrf_desktop/sample.yaml

```diff
@@ -18,7 +18,7 @@ tests:
       - nrf52840gmouse_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     tags: bluetooth ci_build
-  applications.nrf_desktop.zdebug_b0:
+  applications.nrf_desktop.zdebug:
```

Comment:
`applications.nrf_desktop.zdebug:` test should not be duplicated. The old `applications.nrf_desktop.zdebug_b0:` tests are already covered by `applications.nrf_desktop.zdebug:` when b0 enabled configurations are removed.

Reply:
The same with release configurations